openedx/core/djangoapps/edx_global_analytics/tasks.py

```diff
@@ -80,31 +89,59 @@ def count_data():
     # Posts desired data volume to receiving server.
     # Data volume depends on server settings.
     statistics_level = olga_settings.get("STATISTICS_LEVEL")
-    
-    # Platform name.
-    if settings.PLATFORM_NAME:
-        platform_name = settings.PLATFORM_NAME
-    else:
-        platform_name = Site.objects.get_current()
 
-    # Sending instance data
-    # Paranoid level basic data
+    # Paranoid level basic data.
+    courses_amount = len(modulestore().get_courses())
+
+    (active_students_amount_day,
+     active_students_amount_week,
+     active_students_amount_month) = paranoid_level_statistics_bunch()
+
     data = {
-        'active_students_amount': active_students_amount,
+        'active_students_amount_day': active_students_amount_day,
+        'active_students_amount_week': active_students_amount_week,
+        'active_students_amount_month': active_students_amount_month,
         'courses_amount': courses_amount,
         'statistics_level': 'paranoid',
         'secret_token': secret_token
     }
 
-    # Enthusiast level
+    # Enthusiast level (extend Paranoid level)
     if statistics_level == 1:
+        # Get IP address of the platform and convert it to latitude, longitude.
+        ip_data = requests.get('http://freegeoip.net/json')
+        ip_data_json = json.loads(ip_data.text)
+
+        platform_latitude = olga_settings.get("PLATFORM_LATITUDE")
+        platform_longitude = olga_settings.get("PLATFORM_LONGITUDE")
+
+        if platform_latitude and platform_longitude:
```

Comment:
Instead of this `if`, add `ip_data_json['latitude']` and `ip_data_json['longitude']` as a `default` param to `olga_settings.get()`